central/postgres/pruning.go

```diff
@@ -87,21 +87,24 @@ const (
 
 	// Delete the log imbues with old timestamp
 	pruneLogImbues = `DELETE FROM log_imbues WHERE timestamp < now() at time zone 'utc' - INTERVAL '%d MINUTES'`
+
+	pruneAdministrationEvents = `DELETE FROM %[1]s WHERE id in (SELECT id from %[1]s WHERE
```

Comment:
I'm curious, why use a subselect to get the ids when we could simply just say:

`DELETE FROM %[1]s WHERE lastoccurredat < now() at time zone 'utc' - INTERVAL '%d MINUTES`